### PR TITLE
Test cli with credentials (linked to #126)

### PR DESCRIPTION
**Pull request by moul**
Same as #126, but with the scaleway context and credentials

Closes #126

### CODE REVIEW

File: pkg/cli/command.go [hunk 1]
```diff
@@ -68,20 +98,20 @@ func (c *Command) Name() string {
...
-func (c *Command) PrintUsage() {
+// PrintUsage prints a full command usage
+func (c *Command) PrintUsage() error {
 	helpMessage, err := commandHelpMessage(c)
 	if err != nil {
 		logrus.Fatalf("%v", err)
 	}
-	fmt.Fprintf(os.Stderr, "%s\n", helpMessage)
-	os.Exit(1)
+	fmt.Fprintf(c.Streams().Stdout, "%s\n", helpMessage)
```
Comment (QuentinPerez) [inline pkg/cli/command.go]: it should not be c.`Streams().Stderr` ?

Comment (moul) [inline pkg/cli/command.go]: Nope, so we can easily grep the help message

```console
➜  ~  docker run --help >/dev/null
➜  ~  docker run --help 2>/dev/null

Usage: docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Run a command in a new container

  -a, --attach=[]            Attach to STDIN, STDOUT or STDERR
  --add-host=[]              Add a custom host-to-IP mapping (host:ip)
  -c, --cpu-shares=0         CPU shares (relative weight)
  --cap-add=[]               Add Linux capabilities
...
```

Comment (QuentinPerez) [inline pkg/cli/command.go]: Ok 😊

**Comment by moul** [comment]
- Travis result with credentials: https://travis-ci.org/scaleway/scaleway-cli/jobs/76643927
- Travis result without credentials: https://travis-ci.org/scaleway/scaleway-cli/jobs/76643922
- Coveralls result: https://coveralls.io/builds/3366061

**Comment by aimxhaisse** [comment]
lgtm

**Pull request merged**